### PR TITLE
Update review-protocol to 0.7.0

### PR DESCRIPTION
**Pull request by sophie-cluml**
Closes: #313

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -12,6 +12,14 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 - Fixed an issue in the `applyNode` GraphQL API where agents could not be
   properly identified.
 
+## Removed
+
+- The `AgentManager::get_config` method has been removed as part of the new
+  configuration management approach. This aligns with the update to
+  `review-protocol` version 0.7.0.
+
+This version tightens the wording and adds clarity.
```
Comment (sehkone) [inline CHANGELOG.md:21]: I don't think this is what you intended to add.

File: CHANGELOG.md [hunk 2]
```diff
@@ -12,6 +12,14 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 - Fixed an issue in the `applyNode` GraphQL API where agents could not be
   properly identified.
 
+## Removed
```
[the remark this answers is not in the document]
Comment (sophie-cluml) [reply CHANGELOG.md]: Done

File: CHANGELOG.md (hunk 1 above)
Comment (sophie-cluml) [reply CHANGELOG.md]: Done

**Comment by sophie-cluml** [comment]
I am sorry for mistakes @sehkone, I made modifications accordingly.

**Review by sehkone — APPROVED** [review]

**Pull request merged**